src/components/global/base-button.vue

```diff
@@ -0,0 +1,93 @@
+<script>
+    import { defineComponent } from 'vue';
+
+    export default defineComponent({
+        name: 'BaseButton',
+        props: {
+            tag: {
+                type: String,
+                default: 'a',
+            },
+            label: {
+                type: String,
+                required: true,
+            },
+            url: {
+                type: String,
+                default: '#'
+            },
+            colorScheme: {
+                type: String,
+                default: 'redFill',
+            },
+            size: {
+                type: String,
+                required: true,
+            },
+        },
+        emits: ['click'],
+    });
+
+</script>
+
+<template>
+    <component 
+        class="base-button"
+        :is="tag"
+        :url="url"
+        :colorScheme="colorScheme" 
+        :size="size"
+        @click="$emit('click')"
+        >
+        {{label}}
+    </component>
+</template>
+
+<style lang="scss">
+    .base-button {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        border: 2px solid transparent;
+        border-radius: $button-radius;
+        font-weight: $fw-500;
+        line-height: 100%;
```

Comment:
`line-height: 100%` === `line-height: 1`

worth to stick with one notation in the whole app